tsconfig.json

```diff
@@ -24,7 +24,7 @@
     ],
     "paths": {
       "@/*": [
-        "./app/*"
```

Comment:
Should change this to "./src" so we can access to all folder in src by using alias, not just only in app folder.  Remember to add `./` before to avoid error.

Reply:
okay